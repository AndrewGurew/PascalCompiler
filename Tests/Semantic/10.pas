procedure bar(a:double; c:integer);
begin
end

Begin
bar();
end.