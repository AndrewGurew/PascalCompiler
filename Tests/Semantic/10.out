(col: 6, row: 2) - different number of argument in bar